tests/jax/test_custom_call_compute.py

```diff
@@ -0,0 +1,575 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# See LICENSE for license information.
+
+# pylint: disable=missing-module-docstring
+# pylint: disable=missing-function-docstring
+# pylint: disable=missing-class-docstring
```

Comment:
IMO the `tests` directory should either be immune to linting or we should actually comply to the rules. Having lint enabled here but bypassing certain flags does not seem right.